operators/src/pro/meta/quota.rs

```diff
@@ -0,0 +1,38 @@
+use futures::{channel::mpsc::Sender, Future, SinkExt};
+use geoengine_datatypes::identifier;
+use uuid::Uuid;
+
+identifier!(ComputationContext);
+
+/// An Id for a computation used for quota tracking
+#[derive(Debug, Clone, Copy)]
+pub struct ComputationUnit {
+    pub issuer: Uuid,                // TODO: use UserId?
+    pub context: ComputationContext, // TODO: introduce the concept of workflows to the operators crate and use/add it here
+}
+
+#[derive(Clone)]
+pub struct QuotaTracking {
+    quota_sender: Sender<ComputationUnit>,
+    computation: ComputationUnit,
+}
+
+impl QuotaTracking {
+    pub fn new(quota_sender: Sender<ComputationUnit>, computation: ComputationUnit) -> Self {
+        Self {
+            quota_sender,
+            computation,
+        }
+    }
+
+    pub fn work_unit_done(&self) -> impl Future<Output = ()> {
+        let mut sender = self.quota_sender.clone();
+        let computation = self.computation;
+        async move {
+            sender
+                .send(computation)
+                .await
+                .expect("the quota receiver should never close the receiving end of the channel");
```

Comment:
Sure, but shouldn't we rather log an error message? Why should we panic in this case?

Reply:
It is an error that can never happen, that's why we do not need to log anything. The reason why it can never happen is given in the expect message.

Reply:
clippy wants to handle all results. we cold also do `let _ = sender.send` I guess.

Reply:
It is an error that can never happen, that's why we do not need to log anything.

Reply:
In the style of defensive programm: then just do nothing?

let _ = …

Reply:
what's better

```
let _ = bla(); // ignore result because it can never fail
```

or 

```
bla().expect("never fails because...")
```